.github/workflows/trigger_release.yml

```diff
@@ -43,10 +43,12 @@ jobs:
           for d in PolyMC-Windows-*; do
             cd "${d}" || continue
             ARCH="$(echo -n ${d} | cut -d '-' -f 3)"
+            INST="$(echo -n ${d} | grep -o Setup || true)"
             PORT="$(echo -n ${d} | grep -o Portable || true)"
             NAME="PolyMC-Windows-${ARCH}"
             test -z "${PORT}" || NAME="${NAME}-Portable"
-            zip -r -9 "../${NAME}-${{ env.VERSION }}.zip" *
+            test -z "${INST}" || mv PolyMC-*.exe ../${NAME}-Setup-${{ env.VERSION }}.exe
+            test -n "${INST}" || zip -r -9 "../${NAME}-${{ env.VERSION }}.zip" *
```

Comment:
I think we should also provide the zip file in the GH release, even if we don't necessarily link to it on the website.

Reply:
For clarification, do you mean the zip files that have always been made (the regular and portable ones)? They should still be made and attached to the release. If they are not, that was a mistake.

Reply:
Oh, I see. Just a brain fart again. lol

Reply:
No worries, shell syntax with these `test` commands is a little weird 😅